Trappist/src/Promact.Trappist.Web/wwwroot/app/conduct/conduct.service.ts

```diff
@@ -3,7 +3,7 @@ import { HttpService } from '../core/http.service';
 
 @Injectable()
 export class ConductService {
-    private registerTestAttendeeApiUrl: string = '/api/conduct/';
+    private registerTestAttendeeApiUrl: string = '/api/testconduct/';
```

Comment:
I had changed the API route to `api/testconduct` as per your previous review. Please verify the route.